### ISSUE TITLE
Death

### ISSUE THREAD

**Issue by Liareth** [Liareth/SiriusPlanning#30]
How do we handle death?

Right now I'm leaning towards dead players returning as a clone, after a short incubation period. I'd love to clone dead PCs as full-lootable corpses when they die which are persistent (they don't disappear until they are destroyed). There should be some RP justification (tracking chip in clone?) to navigate back to your corpse so that you can grab the loot you lost. Alientech could be bio-locked, so someone could steal your rare alien tech but be unable to do anything with it. Perhaps the incubation period should increase for each death and decrease one death's worth every n time interval.

**Comment by NariKat** [comment]
If we use this concept, a good idea would be that you lose any XP you gained since you last "stored" your clone. We can set up a cloning device in the base camp and even allow players to build and set up others in non-combat areas, and anything you do beyond that point isn't saved, as in the show Altered Carbon. So if you die, your clone will be what you were exactly the last moment you "stored" your memories, thus losing any experience you gained after that.

**Comment by Liareth** [comment]
Yeah, I can get behind that. Lose all exp since you were last in a safe / home zone, plus lose all your items - and you have to go pick them up again, which you are allowed to do because your body has a homing beacon attached to it.
